frontend-ssr/redux/exercise-submissions/exercise-submissions.reducer.ts

```diff
@@ -0,0 +1,54 @@
+import { SubmissionState } from './types';
+import { EXERCISE_SUBMISSIONS } from './exercise-submissions.actions';
+
+const initialState: SubmissionState = {
+  submissions: [],
+};
+export const submissionReducer = (state = initialState, action: { type: string; payload: any;}) => {
```

Comment:
Please add types for the return object, meaning  `: SubmissionState`. This way we would catch errors for bad types.